scripts/herzog

```diff
@@ -1,14 +1,60 @@
 #!/usr/bin/env python
+import argparse
+import json
 import os
 import sys
-import json
 
-pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+from typing import List, Optional
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))  # noqa
 sys.path.insert(0, pkg_root)  # noqa
 
 import herzog
 
 
+def main(args: Optional[List[str]] = None):
+    if args is None:
+        args = sys.argv[1:]
+
+    if len(args) == 1 and args[0] not in ("--help", "-h", "--version", "-v"):
+        # maintain backwards compatibility
+        with open(args[0]) as fh:
+            print(json.dumps(herzog.translate_to_ipynb(fh), indent=2))
+            exit()
+
+    parser = argparse.ArgumentParser(prog="Herzog",
+                                     description="Developer tool to translate notebooks to runnable code and "
+                                                 "back again, allowing notebooks to be written, reviewed, "
+                                                 "and versioned-controlled as normal code.")
```

Comment:
While I appreciate the usage recommendations, I prefer a more purely functional description of Herzog.

Perhaps something like this?:
> Convert Herzog formatted Python scripting to iPython .ipynb files, and back again.